kuttl-test.yaml

```diff
@@ -19,7 +19,7 @@ kind: TestSuite
 reportFormat: JSON
 reportName: kuttl-test-ovn
 namespace: openstack
-timeout: 360
+timeout: 600
```

Comment:
TODO: take a closer look at this. When replicas=3 pods do not appear to terminating gracefully in ovn_tls_enable (cleanup step) and ovn_db_delete (delete-pods step)

Reply:
Cluster cleanup logic was the problem

Reply:
It's not trivial to resolve the cluster cleanup issues on delete, and not directly related to this PR, so for now will just increase the kuttl timeout.

Reply:
what was the issue with cluster cleanup? from a previous run i see:-

`    logger.go:42: 22:50:55 | ovn_db_delete/4-delete-pods | running command: [sh -c oc delete pods -n $NAMESPACE -l service=ovsdbserver-nb
        ]
    logger.go:42: 22:50:56 | ovn_db_delete/4-delete-pods | pod "ovsdbserver-nb-0" deleted
    logger.go:42: 22:50:56 | ovn_db_delete/4-delete-pods | pod "ovsdbserver-nb-1" deleted
    logger.go:42: 22:50:56 | ovn_db_delete/4-delete-pods | pod "ovsdbserver-nb-2" deleted
    logger.go:42: 22:56:55 | ovn_db_delete/4-delete-pods | command failure, skipping 1 additional commands
    case.go:378: failed in step 4-delete-pods
    case.go:380: command "oc delete pods -n $NAMESPACE -l service=ovsdbserver-nb" exceeded 360 sec timeout, context deadline exceeded`

seems delete command just got stuck and with timeout increase it shouldn't help

The current timeout was set to 360s as gracefultimeout is set to 5 minutes.

Reply:
issue is that when the statefulset (or all of the pods) are deleted then pod-0 will exit before pods 1/2 which enter an endless loop waiting to leave the cluster. They are eventually killed but it can take take slightly longer than the current kuttl timeout for the dbcluster delete in the tls_enable test cleanup

Reply:
ack but that still doesn't explain why that much time is required with 5 minute graceful timeout, but as you mentioned over slack the issue was hit locally and logs are no longer available so couldn't see why this case hit. so test improvement can be checked later. As you suggested one option to use --grace-period=30 while kill faster

But yes with tls test it's doing more ops could easily hit currently configured 6 minute timeout so should be fine to increase for now.